infra/src/main/kotlin/fi/fta/geoviite/infra/geometry/GeometryService.kt

```diff
@@ -24,6 +26,27 @@ import java.time.Instant
 
 const val INFRAMODEL_TRANSFORMATION_KEY_PARENT = "error.infra-model.transformation"
 
+val ELEMENT_LISTING_CSV_HEADERS = listOf(
```

Comment:
Mietittiin näitä Tiandun kanssa ja päädyttiin toistaiseksi kovakoodaamaan nämä suomennokset